src/App.vue

```diff
@@ -282,6 +285,7 @@ export default {
         } catch (err) {
           console.log(err) // eslint-disable-line no-console
           this.dashboardUnavailableDialog = true
+          Sentry.captureException(err)
```

Comment:
Why did you decide to log the error here but not elsewhere? Perhaps add a code comment.